action/native_staking_contract_abi.go

```diff
@@ -0,0 +1,35 @@
+package action
+
+import (
+	_ "embed"
+	"strings"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+)
+
+var (
+	// NativeStakingContractJSONABI is the JSON ABI of the native staking contract
+	//go:embed native_staking_contract_abi.json
+	NativeStakingContractJSONABI string
+	// nativeStakingContractABI is the parsed ABI of the native staking contract
+	nativeStakingContractABI abi.ABI
+
+	once sync.Once
+)
+
+func initNativeStakingContractABI() {
+	once.Do(func() {
+		var err error
+		nativeStakingContractABI, err = abi.JSON(strings.NewReader(NativeStakingContractJSONABI))
+		if err != nil {
+			panic("failed to parse native staking contract ABI: " + err.Error())
+		}
+	})
+}
```

Comment:
use function `init` instead, which will only be executed once, such that we can delete `once sync.Once`

Reply:
Initializing directly in the `init()` function will lead to errors because the execution order of init functions in different files within the same package does not follow the order of dependencies. This can result in situations where, for example, the `_candidateRegisterMethod` initialization in candidate_register.go fails.